packages/api-client/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@opendatasoft/api-client",
-    "version": "21.2.0",
+    "version": "21.2.1",
```

Comment:
Lerna does that, probably because I mark the first commit as a feature. It's true that we don't necessarily need a realease before we actually use it in a feature.